src/connections/iList.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package connections;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+/**
+ *
+ * @author Gabriela
+ */
+public class iList implements java.util.List<ListasTablas>{
+       ListasTablas[] campos;
+       ListasTablas[] temp;
+
+    public iList() {
+        campos=new ListasTablas[1];
+    }
+    
+    public iList(ListasTablas y) {
+        campos=new ListasTablas[1];
+        campos[0]=y;
+        u=1;
+        }
+
+    @Override
+    public int size() {        
+        return campos.length;        
+    }
+    @Override
+    public boolean isEmpty() {
+        boolean f=true;
+        f=((size() <= 1) );
+        return f;
+    }
+
+    @Override
+    public boolean contains(Object o) {
+        boolean esta=false;
+        for(Object u:campos){
+        esta=u.equals(o);
```

Comment:
why do you implement `contains` method like this? I think at least it should be:
```java
esta &= u.equals(o);
```